demos/horizontal_text_detection/python/README.md

```diff
@@ -182,3 +182,54 @@ ThreadID:   2; Current FPS:    29.41; Average FPS:    25.70; Average latency:
 ThreadID:   3; Current FPS:    30.30; Average FPS:    25.73; Average latency:   135.65ms
 ...
 ```
+
+## RTSP Client
+
+Build docker image containing rtsp client along with its dependencies
+The rtsp client app needs to have access to RTSP stream to read from and write to.
+
+```bash
+docker build . -t rtsp_client
+```
+
+### Start the client
+
+- Command
+
+```bash
+docker run rtsp_client --help
+usage: rtsp_client.py [-h] [--grpc_address GRPC_ADDRESS]
+                      [--input_stream INPUT_STREAM]
+                      [--output_stream OUTPUT_STREAM]
+                      [--model_name MODEL_NAME] [--width WIDTH]
+                      [--height HEIGHT] [--verbose VERBOSE]
+                      [--input_name INPUT_NAME]
+
+options:
+  -h, --help            show this help message and exit
+  --grpc_address GRPC_ADDRESS
+                        Specify url to grpc service
+  --input_stream INPUT_STREAM
+                        Url of input rtsp stream
+  --output_stream OUTPUT_STREAM
+                        Url of output rtsp stream
+  --model_name MODEL_NAME
+                        Name of the model
+  --width WIDTH         Width of model's input image
+  --height HEIGHT       Height of model's input image
+  --verbose VERBOSE     Should client dump debug information
+  --input_name INPUT_NAME
+                        Name of the model's input
+```
+
+- Usage example
```

Comment:
it would be helpful to add instruction how to create this input stream and how to display the output stream